.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
     working_directory: /go/src/k8s.io/helm
     parallelism: 3
     docker:
-      - image: circleci/golang:1.11.5
+      - image: circleci/golang:1.12.0
```

Comment:
maybe use `circleci/golang:1.12`?

Reply:
I was thinking about it. Didn't know if the explicit patch release number was intentional or not so I opted for the least surprise. :)

I'll go change it as suggested.